user-management-spec.json

```diff
@@ -110,6 +110,53 @@
                 }
             }
         },
+        "/api/v1/refresh-token": {
+            "post": {
+                "tags": [
+                    "User Management"
+                ],
+                "summary": "Refresh the user token",
+                "description": "Refresh the user token",
+                "operationId": "refresh-token",
+                "parameters": [
+                    {
+                        "name": "refresh_token",
+                        "in": "header",
+                        "description": "Valid refresh token",
+                        "required": true,
+                        "schema": {
+                            "type": "string"
+                        }
+                    }
+                ],
+                "responses": {
+                    "400": {
+                        "description": "Invalid request."
+                    },
+                    "200": {
+                        "description": "Successful response - Returns the access token object.",
+                        "content": {
+                            "application/json": {
+                                "schema": {
+                                    "type": "array",
```

Comment:
Is the schema correct here? can you post an example of the response?

Reply:
schema is correct. I removed array wrapper as correction.